### PR TITLE
feat(manage-variable): create clone content & add description field

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
Feature (new)
- Custom variable **clone** feature
- Description field (optional)

Detail
(clone)
- Add CLONE content type. 
- Branching clone CASE in `onMounted` with using `selectedVariable` Props

https://user-images.githubusercontent.com/83635051/226253778-63903354-1426-4181-9dbe-b29154775da2.mov

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue [hunk 1]
```diff
@@ -156,6 +157,10 @@ const handleChangeEditContent = (propertyName: string) => {
     state.selectedVariable = propertyName;
     state.contentType = 'EDIT';
 };
+const handleChangeCloneContent = (propertyName: string) => {
+    state.selectedVariable = propertyName;
+    state.contentType = 'ADD';
```
Comment (yuda110) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:162]: suggestion:
How about `CLONE`?
Then you should check both the `ADD` and `CLONE` cases below.
```typescript
if (state.contentType === 'ADD' || state.contentType === 'CLONE') {
```
But I think this is easier to understand the flow.

**Review by yuda110 — APPROVED** [review]
LGTM!!! I left a comment!

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue (hunk 1 above)
Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:162]: I think so too. But CLONE is also ADD case... hmm.. 
In CLONE case, Heading still have ADD title and ADD form.
So I need to think little more.

Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:162]: I fixed it!

**Pull request merged**